pyflask/manageNeuroconv/info/urls.py

```diff
@@ -0,0 +1,13 @@
+from pathlib import Path
+import json
+
+project_base_path = Path(__file__).parent.parent.parent.parent
+path_config = Path(
+    project_base_path, "paths.config.json"
+)  # NOTE: You're going to have to ensure that this copies over to the Python distribution
+f = path_config.open()
+data = json.load(f)
+stub_save_path = Path(Path.home(), *data["stubs"])
+conversion_save_path = Path(Path.home(), *data["conversions"])
```

Comment:
OK I see; another common convention is for variables like this that are truly global in the sense of the Python side of things, is to all-capitalize them

I'd also recommend adding file or folder specification to it's really clear to someone looking at it what it means; I'm assuming they are directories?

```suggestion
STUB_SAVE_FOLDER_PATH = Path(Path.home(), *data["stubs"])
CONVERSION_SAVE_FOLDER_PATH = Path(Path.home(), *data["conversions"])
```

Reply:
This should be good to go now.